### ISSUE TITLE
gfa command failing due to unitigs.fasta.gz file not being generated during assembly

### ISSUE THREAD

**Issue by lelshura** [GaetanBenoitDev/metaMDBG#12]
Hello,
I am currently using metaMDBG for assembling metagenomic reads. I have encountered an issue where the unitigs.fasta.gz file is not generated during assembly. I have installed metaMDBG by building it from source using conda, following the instructions on the GitHub page. 

I ran the following assembly command:
`metaMDBG asm path/to/output_directory reads_concat.fq.gz -t 52`

The process was completed successfully, and the contigs.fasta.gz file and tmp directory were generated. When looking into the tmp directory, each pass_k# subdirectory contains the assembly_graph.gfa, assembly_graph.gfa.unitigs, and parameters.gz files but no unitigs.fasta.gz file. 

When I tried to run the metaMDBG gfa command
`metaMDBG gfa assemblyDir k --contigpath --readpath`

, I received the following error:
```
metaMDBG gfa ./Scenario_1_concat_reads_metaMDBG 8 --contigpath --readpath
Generating assembly graph for k value: 8 (corresponding sequence length = 1413 bps)
Generating unitig sequences...
Loading unitig sequences...
File not found: ./Scenario_1_concat_reads_metaMDBG/tmp//pass_k8//unitigs.fasta.gz
```
Could you please explain what might be causing this problem and how to generate the unitigs.fasta.gz file correctly?

Thank you

**Comment by GaetanBenoitDev** [comment]
Hello,
This should be a bug from the bioconda version. If you installed from source, the executable is in the following folder ./bin/metaMDBG (in the build folder). So your command becomes: ./bin/metaMDBG gfa ./Scenario_1_concat_reads_metaMDBG 8 --contigpath --readpath

**Comment by GaetanBenoitDev** [comment]
Fixed in v1.0

**Comment by GaetanBenoitDev** [comment]
Fixed in v1.0
